Automattic/wp-calypso
# Readymade Templates: Adjust responsive CSS on generate content page

Related to https://github.com/Automattic/dotcom-forge/issues/8592

## Proposed Changes

* Make some adjustments on the generate content page so it looks better on mobile devices

|Before|After|
|---|---|
|![image](https://github.com/user-attachments/assets/0e056d01-e75f-41bd-834d-c44a216d29e5)|![image](https://github.com/user-attachments/assets/34b69a52-5447-42a2-81a1-22a6831b5a8f)|
|![image](https://github.com/user-attachments/assets/7a297526-a3bd-4a7e-8066-840ea0c8fb90)|![image](https://github.com/user-attachments/assets/fa773abe-00fb-41a0-b586-bd57f504d8de)|
|![image](https://github.com/user-attachments/assets/0bf0b39c-a358-46a7-921c-e7c60f1d1434)|![image](https://github.com/user-attachments/assets/b270a0af-21ad-46bf-a2e2-e169ce1c734d)|

## Testing Instructions



* Open the live preview
* Go to `/patterns`
* Scroll down to the site layouts
* Pick one of the layouts
* Click on `Customize content with AI`
* Wait until loading is done
* With dev tools, enable responsive design
* Check that the content looks good on different mobile sizes

## Pre-merge Checklist
